cmd/release.go

```diff
@@ -179,6 +179,7 @@ func release(cmd *cobra.Command, args []string) error {
 		ReleaseNotes:  releaseNotes,
 		DiscoveryList: listedRelease,
 		Channel:       ReleaseChannelExp, // always experimental release for now
+		Client:        "cli",
```

Comment:
fine for now but I realized we should use an http header instead of the body later for everything relating to meta information about the client.

Reply:
I could change it now as well